src/managed_data/_api.py

```diff
@@ -110,7 +109,7 @@ def describe_database_stats(
     database_id: str,
     *,
     client: Optional[Client] = None,
-):
+) -> dict:
```

Comment:
Same as above -- possibly in a future issue/PR, can we use the `schema` module to type this? This will make it easier to understand the structure of the output.